### ISSUE TITLE
Occasional blackout when switching back from the home screen

### ISSUE THREAD

**Issue by GaucovyAero** [Divested-Mobile/Mull-Fenix#244]
Launcher: crDroid Home (AOSP)
Device: Xiaomi Poco F3 8/256
ROM: crDroid 10.7 (Android 14)

https://github.com/user-attachments/assets/e143b56f-562d-47fe-a31f-5aa1aca01fd0

**Comment by SkewedZeppelin** [comment]
weird. just to confirm, it doesn't happen with other apps?

**Comment by GaucovyAero** [comment]
no, i think i have all the bugs you listed but that isnt a big issue for me, this issue only happens on mull, does not happen on firefox nor fennec f-droid nor iceraven

**Comment by GaucovyAero** [comment]
but this issue is pretty big tbh, cause i just open mull and have to reopen it, sometimes even multiple times to get it working
